packages/app-desktop/gui/utils/loadScript.ts

```diff
@@ -9,7 +9,7 @@ export interface Script {
 	attrs?: Record<string, any>;
 }
 
-export const loadScript = async (script: Script) => {
+export const loadScript = async (script: Script, document: Document) => {
```

Comment:
```suggestion
export const loadScript = async (script: Script, targetDocument: Document) => {
```

I'm not sure whether it's better to shadow the global `document` variable or use a different name. Tradeoffs:
- **Benefit**: Shadowing the global `document` prevents the function's code from accidentally using the wrong `document` object.
- **Drawback**: Shadowing `document` may make it less clear that code in the function is not necessarily referencing the global `document`.